tests/Commands/ToggleDone.test.ts

```diff
@@ -87,13 +95,13 @@ describe('ToggleDone', () => {
         testToggleLine('|', '- |');
     });
 
-    it('should add checkbox to hyphen and space', () => {
-        testToggleLine('|- ', '- [ |] ');
+    it('box to hyphen and space', () => {
```

Comment:
I missed this one...

I like test names that begin with 'should', as it encourages wording that is descriptive when a test fails.

If `box to hyphen and space` failed, I don't think I would understand from the name what the test that failed was doing.